tools/non_central_t_pdf_data.cpp

```diff
@@ -0,0 +1,92 @@
+//  (C) Copyright John Maddock 2024.
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#define BOOST_MATH_MAX_SERIES_ITERATION_POLICY 10000000
+#define BOOST_MATH_USE_MPFR
+
+#include "mp_t.hpp"
+#include <boost/math/special_functions/hypergeometric_1F1.hpp>
+#include <boost/math/distributions/non_central_t.hpp>
+#include <boost/lexical_cast.hpp>
+#include <fstream>
+#include <map>
+#include <boost/math/tools/test_data.hpp>
+#include <boost/random.hpp>
+
+using namespace boost::math::tools;
+using namespace boost::math;
+using namespace std;
+
+struct nc_t_pdf_gen
+{
+   mp_t operator()(mp_t v, mp_t mu, mp_t x)
```

Comment:
Ah, this might be arbitrary precision implementation boost style that generates the test values after all?